contracts/ccip/GovernanceCCIPRelay.sol

```diff
@@ -15,7 +15,7 @@ contract GovernanceCCIPRelay is IGovernanceCCIPRelay {
   IRouterClient public immutable ccipRouter;
```

Comment:
@voith this one is also `immutable` so it should be renamed `CCIP_ROUTER`

Reply:
Lol, I thought you left that intentionally because it makes this call `ccipRouter.ccipSend`. 
I've never seen `CCIP_ROUTER .ccipSend` before. But I'm no style guide expert, so I'll fix this.

Reply:
Usually, it's quite useful to follow that style guide because you visually see immediately whether something can change or not. 

I don't see anything strange on my side to see constant/immutable variables (defined as contracts) that perform function's calls.

Reply:
done